file_paths.md

```diff
@@ -292,6 +294,7 @@ create a `puppet` user or group.
             puppet *                          # ruby bin wrapper
         lib *
             puppet.rb *
+        locale *
```

Comment:
The only prior example I see is in the Ruby install, where it includes a `share` directory. So `share/locale` would be consistent with that and the Linux paths.